modules/planning/scenarios/scenario_manager.cc

```diff
@@ -310,20 +310,18 @@ ScenarioConfig::ScenarioType ScenarioManager::SelectPadMsgScenario(
         return ScenarioConfig::PULL_OVER_EMERGENCY;
       }
       break;
-    // TODO(all): to be added
-    // case DrivingAction::STOP:
-    //  if (FLAGS_) {
-    //    return ScenarioConfig::STOP_EMERGENCY;
-    //  }
-    //  break;
+    case DrivingAction::STOP:
+      // TODO(all): to be added
```

Comment:
Specific assignee is preferred, like `TODO(jiaming, xx): `, other than `all` which means None :)